.circleci/config.yml

```diff
@@ -89,20 +97,52 @@ jobs:
       - restore_cache: *restore_cache
       - run: *install_bundler
       - save_cache: *save_cache
-      - run: *run_lint
       - run: *run_tests
       - store_test_results: *store_test_results
       - store_artifacts: *store_artifacts
 
+  release:
+    description: Release a new version of the Ruby client
+    parameters:
+      version:
+        type: string
+    docker:
+      - *default_docker_ruby_executor
+    steps:
+      - checkout
+      - run: *check_bundler
+      - run: *install_bundler
+      - run:
+          name: Build new gem
+          command: gem build algolia.gemspec
+      - run:
+          name: Publish new gem
+          command: |
+            if [[ -z "$RUBYGEMS_API_KEY" ]]; then echo '$RUBYGEMS_API_KEY is not set'; exit 1; fi
+            gem push --key $RUBYGEMS_API_KEY $(ls algolia-*.gem)
+
 workflows:
   version: 2
   ci:
     jobs:
+      - format:
+          version: '2.7.1'
       - test_ruby:
           matrix:
             parameters:
-              version: ['2.2', '2.3', '2.4', '2.5', '2.6', 'latest']
+              version: ['2.2.10', '2.3.8', '2.4.10', '2.5.8', '2.6.6', '2.7.1']
       - test_jruby:
           matrix:
             parameters:
-              version: ['9', '9.1', '9.2', 'latest']
+              version: ['9.1.17', '9.2.13']
+      - release:
+          version: '2.7.1'
+          requires:
+            - format
+            - test_ruby
+            - test_jruby
+          filters:
```

Comment:
Given that CircleCI treats these filters as `OR` filters, would having only the `tags` filter work for us?

Reply:
Just fixed it in a separate commit (to keep the history if someone is looking at the blame one day).